src/wasm-lib/kcl/src/execution/mod.rs

```diff
@@ -1498,6 +1506,9 @@ pub struct ExecutorSettings {
     /// Should engine store this for replay?
     /// If so, under what name?
     pub replay: Option<String>,
+    /// The directory of the current project.  This is used for resolving import
+    /// paths.  If None is given, the current working directory is used.
+    pub project_directory: Option<PathBuf>,
```

Comment:
Was it intentional to add this back? It seems natural to me that we would need it. But after some refactorings, we were only ever using `None`, so it was removed in #4663.